WcaOnRails/app/jobs/application_job.rb

```diff
@@ -1,4 +1,19 @@
 # frozen_string_literal: true
 
 class ApplicationJob < ActiveJob::Base
+  class DeferJob < StandardError
+    attr_reader :wait_time
+    def initialize(wait_time)
+      super("Delay the job by #{time} s")
```

Comment:
That's right, not sure why I thought that `StandardError`'s constructor requires an argument. Gonna remove this source of confusion!